### PR TITLE
Add Invoke alias for InvokeFunction

### LINKED ISSUE #525: Alias `InvokeFunction` to `Invoke`
Perhaps we could create an alias InvokeTransaction for InvokeFunction?

_Originally posted by @Solpatium in https://github.com/software-mansion/starknet.py/pull/499#discussion_r1026413147_

### PR DESCRIPTION
**Pull request by drknzz**
Closes #525 

- Add `Invoke` alias
- Ignore pylint's `unused-import` for backward compatibility

- Fix error `"stubPath /.../starknet.py/typings is not a valid directory."` when running typechecker

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -101,3 +101,4 @@ exclude = [
     "**/__pycache__",
     "starknet_py/tests/e2e/docs",
 ]
+stubPath = ""
```
Comment (cptartur) [inline pyproject.toml:104]: Why is that needed?

File: starknet_py/net/account/account_client.py [hunk 2]
```diff
@@ -636,7 +633,7 @@ def parse_call(
     return [calldata, entire_calldata]
 
 
-def make_invoke_function_by_version(
+def make_invoke_by_version(
```
[the remark this answers is not in the document]
Comment (cptartur) [reply starknet_py/net/account/account_client.py:636]: I don't think so this should be consider a private function anyways.

**Review by cptartur — COMMENTED** [review]
Looks good.

I wonder if there's a way to add some kind of deprecation warning if someone uses `InvokeFunction` but that probably would be complicated.

File: pyproject.toml (hunk 1 above)
Comment (drknzz) [reply pyproject.toml:104]: The 3rd point in PR description:

- Fix error `"stubPath /.../starknet.py/typings is not a valid directory."` when running typechecker

The error does nothing bad, just annoying to see.

Comment (cptartur) [reply pyproject.toml:104]: Interesting, I never noticed this error but if it fixes it, I guess we can keep it.

Comment (cptartur) [reply pyproject.toml:104]: cc @Solpatium @war-in

Comment (drknzz) [reply pyproject.toml:104]: https://github.com/microsoft/pyright/issues/777

**Comment by drknzz** [comment]
> I wonder if there's a way to add some kind of deprecation warning if someone uses `InvokeFunction` but that probably would be complicated.

This exists: https://peps.python.org/pep-0562/#rationale

File: pyproject.toml (hunk 1 above)
Comment (war-in) [reply pyproject.toml:104]: It's OK for me but please add a comment about why we need that

**Review by war-in — APPROVED** [review]
All good, just one question. Should we create an issue to remove the `InvokeFunction,  # backward compatibility` one day?

File: starknet_py/net/account/account_client.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (Solpatium) [reply starknet_py/net/account/account_client.py:636]: Please prefix it with `_` to mark that it is private.

File: starknet_py/net/full_node_client.py [hunk 3]
```diff
@@ -7,12 +7,13 @@
...
 from starknet_py.net.client_errors import ClientError
-from starknet_py.net.client_models import (
+from starknet_py.net.client_models import (  # pylint: disable=unused-import
     SentTransactionResponse,
     TransactionReceipt,
     BlockStateUpdate,
     StarknetBlock,
-    InvokeFunction,
+    InvokeFunction,  # backward compatibility
+    Invoke,
```
Comment (Solpatium) [inline starknet_py/net/full_node_client.py:16]: I am not sure if we need to leave `InvokeFunction` imports in every place. If someone imported it from a wrong place they should adjust their import paths. What do you guys think?

Comment (Solpatium) [reply starknet_py/net/full_node_client.py:16]: @cptartur @war-in

Comment (cptartur) [reply starknet_py/net/full_node_client.py:16]: I think it makes sense to do remove them.

**Review by Solpatium — APPROVED** [review]
🔥
